src/pages/dashboard/LatestPublications.tsx

```diff
@@ -44,7 +43,8 @@ const LatestPublications: FC = () => {
                 }
                 secondaryComponent={
                   <Typography component="span">
-                    {new Date(publication.modifiedDate).toLocaleDateString()} - {publication.owner}
+                    {publication.modifiedDate && new Date(publication.modifiedDate).toLocaleDateString()} -
+                    {publication.owner}
```

Comment:
kan `modifiedDate` være `undefined`? i så fall blir det " - Navn Navnesen" her nå

Reply:
åja, det er siden det er ny type ja, må kanskje vente til Search API er ferdig med å konkludere da

Reply:
egentlig ikke, men siden jeg slo sammen PublicationListItem og SearchResult-typene, så brukte jeg `Partial`. Men er vel mer rett å bruke Pick og lage ny type. Men da blir det to typer igjen da 😝

Reply:
LatestPublications har ikke contributors, men det har SearchResults

Reply:
lagde LatestPublication-type. Følte det ble mest rett.